src/analysis/souffle/taint.dl

```diff
@@ -78,4 +81,6 @@ mayHaveTag(base, tag) :-
   mayHaveTag(member, tag),
   isMemberOf(base, member).
 
+.output checkHasTag
```

Comment:
```suggestion
.output checkHasTag(IO=stdout, delimiter=",")
```

The reason why I suggest the above is that the default behavior by Souffle is to print the output to a file named similarly to the output relation in the current directory. I find that behavior rather surprising, especially when I've run Souffle in interpreter mode to get a sense of what it's doing when I have a test failure. I think that outputting to `stdout` is generally much less surprising; I also think that the `,` delimiter is much more readable than a `<tab>`.

Reply:
You can dump to stdout by passing `-D-` on the souffle command line. Specifying `","` as a delimiter is a good idea, but I am going to abandon this PR completely as we should probably do something along the lines of https://github.com/google-research/raksha/blob/main/src/analysis/souffle/fact_test_helper.dl